### ISSUE TITLE
Connecting to Kubeflow JupyterLab Notebook Kernel

### ISSUE THREAD

**Issue by hendrikheil** [kubeflow/kubeflow#7475]
/kind question

**Question:**
Is there currently any way to connect to a JupyterLab Notebook Kernel running in Kubeflow?
We're trying to connect a VSCode Jupyter extension to use a Kubeflow notebook, but are dealing with authentication issues.

I think this is because of oauth2-proxy being responsible for auth. I saw the base notebook image configures the lab to allow any remote connections with an empty token / password, but it doubt VSCode even gets that far in the first place.

So I'm wondering if anybody has successfully connected VSCode to a Kubeflow Notebook Kernel?

**Comment by juliusvonkohout** [comment]
I think that is for kubeflow/manifests. And yes its possible.

You can join the CNCF Slack and access our meetings at the [Kubeflow Community](https://www.kubeflow.org/docs/about/community/) website. Our channel on the CNCF Slack is here [#kubeflow-platform](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2). You can also find there our [biweekly meetings](https://bit.ly/kf-wg-manifests-meet), including the commentable [Agenda](https://bit.ly/kf-wg-manifests-notes).

/close

**Comment by RodrigoRP** [comment]
Hi @hendrikheil , I saw your question from February about connecting VS Code to a Kubeflow Notebook Kernel. Did you manage to find a solution or workaround for the OAuth2 authentication issue?

**Comment by hendrikheil** [comment]
No, unfortunately we never did and eventually moved away from kubeflow to something smaller that also worked well for us
